cmd/zfs/zfs_main.c

```diff
@@ -6868,7 +6868,7 @@ print_holds(boolean_t scripted, int nwidth, int tagwidth, nvlist_t *nvl,
 				if (parsable) {
 					(void) printf("%-*s  %-*s  %ld\n",
 					    nwidth, zname, tagwidth,
-					    tagname, time);
+					    tagname, (unsigned long)time);
```

Comment:
You want `(long_t)` here since, the format specifier is signed.  Or switch it to `%lu`.